### DISCUSSION TITLE
who have hamc body demo

### DISCUSSION THREAD

**Discussion by baiuu** [apache/apisix#6159] [Q&A] [answered]
who have hamc body demo 
i can't realize

**Comment by leslie-tsang** [comment, marked as answer]
Please refer to [hmac-auth3.t](https://github.com/apache/apisix/blob/master/t/plugin/hmac-auth3.t#L117-L173)

**Reply by baiuu** [reply]
thanks

**Reply by leslie-tsang** [reply]
Hello there, Please close the discussion if problem has been solved.
